src/components/icon-dropdown/index.js

```diff
@@ -74,7 +75,7 @@ class Dropdown extends Component {
                     />
                 {visible &&
                     <div data-focus='dropdown-menu' data-position={openDirection} ref='dropdown'>
-                        {operationList.map(({label, action}, idx) => (<div key={idx} data-role='dropdown-item' onClick={this._operationActionWrapper(action)}>{label}</div>))}
+                        {operationList.map(({label, action}, idx) => (<div key={idx} data-role='dropdown-item' onClick={this._operationActionWrapper(action)}>{i18n.t(label)}</div>))}
```

Comment:
Almost there ! 
For `i18n`, you can't rely on the global variable `window.i18n`. It's considered as unstable.
But you did well : you used the `i18nMixin`. This will set a `i18n` method in your class, so instead of using `i18n.t()` you might want to use `this.i18n()`.
I let you change it and test, then I'll merge it.